mgr.go

```diff
@@ -135,9 +137,14 @@ func newSysboxMgr(ctx *cli.Context) (*SysboxMgr, error) {
 		return nil, fmt.Errorf("failed to setup shiftfs mgr: %v", err)
 	}
 
-	linuxHeaderMounts, err := getLinuxHeaderMounts()
+	sysDistro, err := libutils.GetDistro()
```

Comment:
Is this the host distro? If so, let's rename sysDistro -> hostDistro to avoid confusion with "sys containers".

Reply:
mmm ... good point.